AODN/AODN-WAVE-DM/MHL/process_MHLwave_from_txt.py

```diff
@@ -11,13 +11,13 @@
 import pandas
 import numpy as np
 from netCDF4 import Dataset, date2num
-from generate_netcdf_att import generate_netcdf_att
+from python.generate_netcdf_att import generate_netcdf_att
```

Comment:
This ```python.``` should be removed

Reply:
That's the only way it worked on my machine. I am not changing it

Reply:
on your machine yes maybe, but that's not the way sorry. if this was used on RC or prod, it shouldn't have python.